avpipe.c

```diff
@@ -387,7 +390,7 @@ udp_in_closer(
 
     int fd = *((int64_t *)inctx->opaque);
     int sockfd = *((int *)((int64_t *)inctx->opaque+1));
-    elv_dbg("IN CLOSE UDP fd=%d, sockfd=%d, url=%s\n", fd, sockfd, inctx->url);
+    elv_dbg("IN CLOSE UDP fd=%d, sockfd=%d, url=%s\n", fd, sockfd, inctx->url ? inctx->url : "bogus.mp4");
```

Comment:
This seems to be the issue for inctx not being null, is that it will not necessarily have a URL.